specification/servicenetworking/ServiceNetworking.Management/main.tsp

```diff
@@ -147,6 +233,13 @@ interface FrontendsInterface
   extends ResourceInstanceOperations<Frontend, FrontendProperties>,
     ResourceListByParent<Frontend> {}
 
+@added(Versions.v2024_05_01_preview)
+@removed(Versions.v2023_11_01)
+@armResourceOperations
+interface SecurityPoliciesInterface
```

Comment:
Can you instead of using ResourceInstanceOperations interface,
use individual operations in that interface? See playground examples. https://azure.github.io/typespec-azure/playground?options=%7B%22linterRuleSet%22%3A%7B%22extends%22%3A%5B%22%40azure-tools%2Ftypespec-azure-resource-manager%2Fall%22%5D%7D%7D&sample=Azure+Resource+Manager+framework